### DISCUSSION TITLE
You are first result in Google

### DISCUSSION THREAD

**Discussion by tercerapersona** [nelsonjchen/github-wiki-see-rs#152] [Q&A] [answered]
hey, thanks for push forward on indexing wikis! I wonder how you got ranked so quickly, without having any backlinks to your site?

**Comment by nelsonjchen** [comment, marked as answer]
I think just being useful and having very lightweight pages. I am content with being on page 2 or worse. I think people keep clicking the results and don't return to the search results so some float to the top.
